std/uni.d

```diff
@@ -9108,7 +9108,15 @@ bool isMark(dchar c)
 @safe pure nothrow @nogc
 bool isNumber(dchar c)
 {
-    return numberTrie[c];
+    // optimization
+    if (c < 0xAA)
```

Comment:
No it IS slower because all of non-digit ascii chars will fall through to trie case (slow path) which is quite a common case to optimize for.